### ISSUE TITLE
SSR `nuxtSiteConfig` on `/locale`

### ISSUE THREAD

**Issue by luca-smartpricing** [harlan-zw/nuxt-seo#199]
### Describe the bug

If I use `nuxtSiteConfig` on `/locale` with i18n, it doesn't work on SSR.

I found the description here: https://www.nuxtseo.com/site-config/integrations/i18n#usage

If i inspect the code i can see the meta: 
`<meta name="description" content="EN - default description">`
but if i inspect the source code this meta doesn't exist

### Reproduction

https://stackblitz.com/edit/github-v9xq4g?file=app.vue

### System / Nuxt Info

```Shell
- Operating System: Darwin
- Node Version:     v20.11.0
- Nuxt Version:     3.10.3
- CLI Version:      3.10.1
- Nitro Version:    2.8.1
- Package Manager:  yarn@4.0.1
- Builder:          -
- User Config:      telemetry, nitro, vite, experimental, runtimeConfig, css, modules, site, sitemap, ogImage, schemaOrg, i18n, googleFonts, gtm, tailwindcss, prismic, postcss, devtools
- Runtime Modules:  @nuxtjs/google-fonts@3.1.3, @zadigetvoltaire/nuxt-gtm@0.0.13, @nuxtjs/tailwindcss@6.11.4, @vueuse/nuxt@10.8.0, @nuxtjs/prismic@3.1.0, @nuxtjs/i18n@8.1.1, @nuxtjs/seo@2.0.0-rc.8, @nuxt/image@1.3.0, nuxt-icon@0.6.8
- Build Modules:    -
```

**Comment by harlan-zw** [comment]
Thanks, looks like an issue when the Nuxt SEO plugin runs, not an issue with Nuxt Site Config itself, have been the issue.

**Comment by harlan-zw** [comment]
I've pushed up a fix in 2.0.0-rc.9, seems to fix the reproduction. Thanks again :+1:

**Comment by luca-smartpricing** [comment]
Thank you! Can't wait to update
